spring/boot-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaAutoConfigurationTest.java

```diff
@@ -168,6 +179,13 @@ public AggregatedHttpResponse get() {
         public AggregatedHttpResponse getV2() {
             throw new IllegalArgumentException();
         }
+
+        @Post("/post")
+        @ProducesJson
+        public AggregatedHttpResponse post(@RequestObject JsonNode jsonNode) throws JsonProcessingException {
```

Comment:
I guess you can just return the JsonNode as-is, without any conversion because Armeria will do that for you.

Reply:
Thank you for review! a579b0a